.laminas-ci.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ignore_php_platform_requirements": {
+    "8.1": true
```

Comment:
Is this still needed with the new dependency releases?

Reply:
Let's keep it for now